layers/LAYERS.org

```diff
@@ -67,64 +77,81 @@
 - [[file:+intl/chinese/README.org][chinese]]
 - [[file:+intl/keyboard-layout/README.org][keyboard-layout]]
 
-* Programming and markup languages
-- [[file:+lang/agda/README.org][agda]]
-- [[file:+lang/asciidoc/README.org][asciidoc]]
-- [[file:+lang/asm/README.org][asm]]
+* General-purpose and domain-specific languages
```

Comment:
Let's call it simply `Programming Languages` and have a `Markup Languages` at the same level.

Reply:
Done. `Markup languages` now top-level entry as `Programming Languages`.